src/components/MoonpayModal/MoonpayModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CustomModal } from 'components';
+
+interface MoonpayModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const MoonpayModal: React.FC<MoonpayModalProps> = ({ open, onClose }) => {
+  return (
+    <CustomModal
+      open={open}
+      onClose={onClose}
+      background='#fff'
+      overflow='hidden'
+    >
+      <div style={{ height: '100%', width: '100%', overflowY: 'auto' }}>
+        <iframe
+          title='moonpay'
+          allow='accelerometer; autoplay; camera; gyroscope; payment'
```

Comment:
Why do we need to allow camera? Probably kyc? Add a comment please in a seperate pr